src/scripts/calibration_analyses/analysis_scripts/analysis_demography_calibrations.py

```diff
@@ -87,7 +87,7 @@ def apply(results_folder: Path, output_folder: Path, resourcefilepath: Path = No
     # Load Data: WPP_Annual
     wpp_ann = pd.read_csv(Path(resourcefilepath) / "demography" / "ResourceFile_Pop_Annual_WPP.csv")
     wpp_ann['Age_Grp'] = wpp_ann['Age_Grp'].astype(make_age_grp_types())
-    wpp_ann_total = wpp_ann.groupby(['Year']).sum().sum(axis=1)
+    wpp_ann_total = wpp_ann.groupby(['Year'])['Count'].sum()
```

Comment:
`wpp_ann.groupby(['Year'])['Count'].sum()` isn't exactly the same as the old `wpp_ann.groupby(['Year']).sum().sum(axis=1)`, that'd be `wpp_ann.groupby(['Year'])[['Age', 'Count']].sum().sum(axis=1)`, but it looks suspicious one wants to sum Age _and_ Count columns together?

Reply:
Agreed this looks a bit odd - running with Pandas v1.5 we get that `wpp_ann.groupby(['Year']).sum()` outputs

```
        Age       Count
Year
1950  10100   2953871.0
1951  10100   3011789.0
1952  10100   3072386.0
1953  10100   3135689.0
1954  10100   3201722.0
...     ...         ...
2096  10100  65108710.0
2097  10100  65487105.0
2098  10100  65854995.0
2099  10100  66212402.0
2100  10100  66559386.0
```

while `wpp_ann.groupby(['Year']).sum().sum(axis=1)` outputs 

```
Year
1950     2963971.0
1951     3021889.0
1952     3082486.0
1953     3145789.0
1954     3211822.0
           ...
2096    65118810.0
2097    65497205.0
2098    65865095.0
2099    66222502.0
2100    66569486.0
Length: 151, dtype: float64
```

which as @giordano says seems to be summing up `Age` and `Count` columns.  @tbhallett is this intentional?

Reply:
This is good spot. The old versions look like an error and the new version seems correct to me!